homeassistant/components/api/__init__.py

```diff
@@ -118,8 +118,11 @@ def get(self, request: web.Request) -> web.Response:
         Home Assistant core is running. Its primary use case is for supervisor
         to check if Home Assistant is running.
         """
-        hass = request.app[KEY_HASS]
-        return self.json({"state": hass.state.value})
+        hass: HomeAssistant = request.app[KEY_HASS]
```

Comment:
The explicit type annotation shouldn't be needed when using an aiohttp AppKey.

https://github.com/home-assistant/core/blob/f135d3d16cf0dee692c9092a4f9332b50ff5d98d/homeassistant/helpers/http.py#L37